plugins/vtd/conanfile.py

```diff
@@ -65,5 +65,13 @@ def package(self):
         cmake.install()
         self.copy("vtd-launch", dst="bin", src=f"{self.source_folder}/bin")
 
+    def package_id(self):
+        # Changes in a dependency's package_id need to be accounted for since
+        # the package statically links against vtd-object-lib.
+        self.info.requires["cloe-models"].full_package_mode()
```

Comment:
If we do this here, do we need to do this in other plugins too?

Reply:
I think the special situation of the vtd plugin is the static library that is built to have the object files available also for testing. This is not done in other plugins as far as I know.